src/RESTController.js

```diff
@@ -152,6 +154,9 @@ const RESTController = {
         headers['User-Agent'] = 'Parse/' + CoreManager.get('VERSION') +
           ' (NodeJS ' + process.versions.node + ')';
       }
+      if (CoreManager.get('IDEMPOTENCY')) {
```

Comment:
Since we only send the header for POST, PUT, I think this should be something like `if (CoreManager.get('IDEMPOTENCY') && ['POST', 'PUT'].includes(method)) {`